### PR TITLE
Create skin type test info page

### PR DESCRIPTION
**Pull request by dpionk**
Created skin type test info page along with routes

### CODE REVIEW

File: skin-resque-frontend/src/pages/SkintypeTest/Info/Info.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Link } from 'react-router-dom';
+import Button from '../../../components/Button/Button';
+import Text from '../../../components/Text/Text';
+import './Info.scss';
+
+function Info() {
+	return (
+		<div className='info-container'>
+			<div className='info'>
```
Comment (kbojarowska) [inline skin-resque-frontend/src/pages/SkintypeTest/Info/Info.js:9]: This part should be justified

File: skin-resque-frontend/src/pages/SkintypeTest/SkintypeTestPage/SkintypeTestPage.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Heading from '../../../components/Heading/Heading';
+import './SkintypeTestPage.scss';
+import Info from '../Info/Info';
+
+function SkintypeTestPage() {
+
+	return (
+		<div className='skintype-test-page'>
+			<div className='page-title'>
+				<Heading size='large'>Skintype Test</Heading>
```
Comment (kbojarowska) [inline skin-resque-frontend/src/pages/SkintypeTest/SkintypeTestPage/SkintypeTestPage.js:10]: test should be lower

**Review by kbojarowska — CHANGES_REQUESTED** [review]
Why do you want to seperate one page info two parts? Idk if we need it

**Review by MrSpaceMan1 — APPROVED** [review]

**Comment by dpionk** [comment]
> Why do you want to seperate one page info two parts? Idk if we need it

Because I already implemented  questions page which will be rendering on SkinTypeTestPage component. I needed to merge these changes in order to add the questions part next

**Comment by dpionk** [comment]
@kbojarowska  I updated the code

**Review by kbojarowska — APPROVED** [review]

**Pull request merged**